frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -204,6 +233,37 @@ describe("Test PollingStationChoiceForm", () => {
       // Check if the error message is visible
       expect(screen.getByText("Geen stembureaus gevonden")).toBeVisible();
     });
+
+    test("Second data entry has correct link", async () => {
+      overrideOnce("get", "/api/elections/1", 200, electionDetailsMockResponse);
+      overrideOnce("get", "/api/elections/1/status", 200, {
+        statuses: [
+          {
+            id: 1,
+            status: "second_entry",
+          },
+          {
+            id: 2,
+            status: "definitive",
+          },
+        ],
+      } satisfies ElectionStatusResponse);
+
+      const mockNavigate = vi.fn();
+      vi.spyOn(router, "useNavigate").mockImplementation(() => mockNavigate);
+
+      renderPollingStationChoicePage();
+
+      // Open the polling station list
+      const user = userEvent.setup();
+      const openPollingStationList = await screen.findByTestId("openPollingStationList");
+      await user.click(openPollingStationList);
+
+      // Click polling station 33 and check if the link is correct
+      const pollingStationList = await screen.findByTestId("polling_station_list");
+      await userEvent.click(within(pollingStationList).getByText("33"));
+      expect(mockNavigate).toHaveBeenCalledWith("/elections/1/data-entry/1/2");
```

Comment:
I think it's clearer to not have 2 numbers following each other in a URL. Maybe add the word `/entry/` in between? Or change the URL to `/elections/{election_id}/polling_station/{polling_station_id}/data-entry/{entry_number}` for example

Reply:
Created #697 to discuss this further.